contrib/completion/zsh/_docker

```diff
@@ -138,6 +138,14 @@ __docker_containers_names() {
     __docker_get_containers all names "$@"
 }
 
+__docker_plugins() {
+    [[ $PREFIX = -* ]] && return 1
+    integer ret=1
+    plugins=(${(f)"$(_call_program commands docker $docker_options info | sed -n "/^Plugins/,/^[^ ]/s/ $1: //p")"})
```

Comment:
@vincentbernat Hey, maybe you can help me on this, I'm not as good as you for black-magic substitution etc.
When there is more than one plugin, it doesn't split correctly into a array. For example, I get `-f driver=bridge\ null\ host` instead of multiple values during the auto-complete.